lib/runner/extensions/event.command.js

```diff
@@ -111,6 +111,7 @@ module.exports = {
         }
 
         sandbox.createContext(_.merge({
+            timeout: _(run.options.timeout).pick(['script', 'global']).values().min()
```

Comment:
fancy! :-)

Reply:
I was probably in the Python one liner mode here. 😄